front/components/assistant/details/AssistantUsageSection.tsx

```diff
@@ -0,0 +1,71 @@
+import type {
+  LightAgentConfigurationType,
+  LightWorkspaceType,
+} from "@dust-tt/types";
+import { useMemo } from "react";
+
+import { assistantUsageMessage } from "@app/components/assistant/Usage";
+import { useAgentUsage } from "@app/lib/swr/assistants";
+import { timeAgoFrom } from "@app/lib/utils";
+
+interface AssistantUsageSectionProps {
+  agentConfiguration: LightAgentConfigurationType;
+  owner: LightWorkspaceType;
+}
```

Comment:
Question: what did we agree on regarding sharing props between components?

Reply:
We can't actually since one of them is `LightAgentConfigurationType` and the other is `AgentConfigurationType`.